### ISSUE TITLE
Offline/online layer buttons remain disabled

### ISSUE THREAD

**Issue by mapcentia** [sashuk/vidi#189]
On this installation offline/online layer buttons remain disabled:
https://map.gc2.io/app/mydb/public/#osm/10/10.6561/56.377/v:public.test1

But on this its working:
https://dev.geofyn.dk/app/test/public/#osm/13/10.4012/55.5076/v:public.registrering

Its not obvious whats wrong.

![image](https://user-images.githubusercontent.com/3850918/48674515-73a4f600-eb4d-11e8-9bee-33eb85254785.png)

**Comment by sashuk** [comment]
@mapcentia https://vidi.alexshumilov.ru/app/mydb/public/#osm/10/10.6561/56.377/v:public.test1 seems to work fine, however, seems like the https://map.gc2.io/app/mydb/public/#osm/10/10.6561/56.377/v:public.test1 version is not able to create caches from service worker (both version have `localforage@1.7.3`).

The error handling in service worker iterations with cache is more verbose now (https://github.com/sashuk/vidi/commit/ed81825484c0c9d759c731368f5de755e6e18c08), could you please update and rebuild the service worker? Also could you please provide temporary access (user and password for testing the #189)?

**Comment by mapcentia** [comment]
@sashuk Its updated. 
User and pw for GC2 Admin?

**Comment by sashuk** [comment]
@mapcentia Thank you very much! 

Yes, please, if it possible, as my GC2 user is not able to commit changes to the dataset.

**Comment by mapcentia** [comment]
@sashuk The info is send over email

**Comment by sashuk** [comment]
@mapcentia Thanks!

Seems like the problem is because of 

```

let urlsIgnoredForCaching = [{
    regExp: true,
    requested: 'bam.nr-data.net'
}, {
    regExp: true,
    requested: 'gc2.io/api'
}, {
    regExp: true,
    requested: '/version.json'
}, {
    regExp: true,
    requested: 'geocloud.envirogissolutions.co.za/api'
}, {
    regExp: true,
    requested: 'geofyn.mapcentia.com/api'
}];
```

, so the service worker is not even trying to cache requests to `*gc2.io/api*`. Currently investigating.

**Comment by mapcentia** [comment]
@sashuk I've added the `requested: 'gc2.io/api'` and I guess its because Vidi is on map.gc2.io?

**Comment by mapcentia** [comment]
@sashuk is it possible to add the full URL for gc2.io ? Like https://gc2.io/api and avoid sub-domains bypassing the cache?

**Comment by sashuk** [comment]
@mapcentia Please try to build the service worker again, the rule was added

**Comment by mapcentia** [comment]
@sashuk Nice, its working now. It also fixed #190 Thanks!

**Comment by sashuk** [comment]
@mapcentia Happy to hear that!
